lumu/info.json

```diff
@@ -0,0 +1,461 @@
+      },
+      {
+        "title": "Company Key",
+        "type": "password",
+        "name": "company_key",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "tooltip": "Specify the API Key that you will use to access the LUMU REST API.",
+        "description": "Specify the API Key that you will use to access the LUMU REST API."
+      },
+      {
+        "title": "Verify SSL",
+        "name": "verify_ssl",
+        "type": "checkbox",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "value": true,
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified or not.By default, this option is set as True.",
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not.By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_incidents",
+      "title": "Get Incidents",
+      "description": "Retrieves a paginated list of incidents for the company. The items are listed by the most recent.",
+      "category": "investigation",
+      "annotation": "get_incidents",
+      "enabled": true,
+      "output_schema": {
+        "items": [
+          {
+            "id": "",
+            "timestamp": "",
+            "statusTimestamp": "",
+            "status": "",
+            "contacts": "",
+            "adversaries": [],
+            "adversaryTypes": [],
+            "labelDistribution": {
+              "17": ""
+            },
+            "totalEndpoints": "",
+            "lastContact": "",
+            "unread": ""
+          }
+        ],
+        "paginationInfo": {
+          "page": "",
+          "items": ""
+        }
+      },
+      "parameters": [
+        {
+          "title": "From Date",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "datetime",
+          "name": "fromDate",
+          "description": "Specify the Search start date. The default value is 7 days before the current date. Example: 2024-04-01T14:40:14.939Z",
+          "tooltip": "Specify the Search start date. The default value is 7 days before the current date. Example: 2024-04-01T14:40:14.939Z"
+        },
+        {
+          "title": "To Date",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "datetime",
+          "name": "toDate",
+          "description": "Specify the Search end date. The default value is the current date. Example: 2021-04-07T14:40:14.939Z",
+          "tooltip": "Specify the Search end date. The default value is the current date. Example: 2021-04-07T14:40:14.939Z"
+        },
+        {
+          "title": "Incident Status",
+          "name": "status",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "multiselect",
+          "options": [
```

Comment:
Pls make options as Open, Muted and Closed and handle it in operation using lower()